doc/nrf/device_guides/working_with_nrf/nrf70/developing/sap.rst

```diff
@@ -0,0 +1,122 @@
+.. _nRF70_soft_ap_mode:
+
+SoftAP mode
+###########
+
+.. contents::
+   :local:
+   :depth: 2
+
+The nRF70 Series device supports :term:`Software-enabled Access Point (SoftAP or SAP)` mode.
+This mode allows a device to operate as a virtual router or temporary access point (AP) and accept connections from other Wi-Fi station devices.
+SoftAP mode is typically used in scenarios where a Wi-Fi device wishes to share its internet connection with nearby station devices.
+For instance, smartphone Wi-Fi hotspots utilize SoftAP mode.
+The smartphone acts as a virtual router, allowing other station devices to connect to the internet through it.
+Alternatively, SoftAP mode can be used to securely provision Wi-Fi devices into an access point network.
+
+nRF70 Series devices support Wi-Fi provisioning using the SoftAP mode as one of the provisioning methods.
+
+Supported functionality and limitations
+***************************************
+
+The SAP mode in the nRF70 Series devices provides SoftAP support with the following limitations:
+
+  * A network interface can operate in either SAP mode or Station mode, but not in both modes simultaneously.
+  * SAP mode only supports the Wi-Fi 4 (IEEE 802.11n) mode of operation.
+  * The SAP mode currently supports a single station device connection.
+
+      However, the implementation provides an :ref:`experimental <software_maturity>` support for up to eight station connections.
+
+  * Only Wi-Fi personal security (WPA2-PSK) is supported.
```

Comment:
Formatting issue https://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-13562/nrf/device_guides/working_with_nrf/nrf70/developing/sap.html#supported-functionality-and-limitations
```suggestion
  * A network interface can operate in either SAP mode or Station mode, but not in both modes simultaneously.
  * SAP mode only supports the Wi-Fi 4 (IEEE 802.11n) mode of operation.
  * The SAP mode currently supports a single station device connection.
     However, the implementation provides an :ref:`experimental <software_maturity>` support for up to eight station connections.
 * Only Wi-Fi personal security (WPA2-PSK) is supported.
```

Reply:
if I don't put a newline, the previous line is highlighted in `bold`